jvb/src/main/kotlin/org/jitsi/videobridge/EndpointK.kt

```diff
@@ -268,6 +269,20 @@ class EndpointK @JvmOverloads constructor(
 
     override fun requestKeyframe(mediaSsrc: Long) = _transceiver.requestKeyFrame(mediaSsrc)
 
+    override fun isOversending(): Boolean = bitrateController.isOversending()
+
+    override fun setSelectedEndpoints(selectedEndpoints: MutableList<String>) {
+        bitrateController.setSelectedEndpoints(selectedEndpoints)
+    }
+
+    override fun setMaxFrameHeight(maxFrameHeight: Int) {
+        bitrateController.setMaxFrameHeight(maxFrameHeight)
+    }
+
+    override fun setBandwidthAllocationSettings(message: ReceiverVideoConstraintsMessage) {
+        bitrateController.setBandwidthAllocationSettings(message)
+    }
+
```

Comment:
These three could use Kotlin `=` notation.

Reply:
I changed `setMaxFrameHeight` and `setBandwidthAllocationSettings` but not sure which third one you meant.

Reply:
`setSelectedEndpoints`.

Reply:
K done